examples/run_squad.py

```diff
@@ -52,7 +52,7 @@
 logger = logging.getLogger(__name__)
 
 ALL_MODELS = sum((tuple(conf.pretrained_config_archive_map.keys()) \
-                  for conf in (BertConfig, XLNetConfig, XLMConfig)), ())
+                  for conf in (BertConfig, RobertaConfig, XLNetConfig, XLMConfig)), ())
```

Comment:
Do we need to add DistilBertConfig here?

Reply:
added